fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -415,7 +415,11 @@ private void initDefaultJobProperties() {
                 timeout = Config.broker_load_default_timeout_second;
                 break;
             case INSERT:
-                timeout = Config.insert_load_default_timeout_second;
+                if (ConnectContext.get() != null) {
```

Comment:
Generally LGTM
But it would be better if the code style can be optimized. Similar to this code, suggest remove redundant if..else
```suggestion
timeout = Optional.ofNullable(ConnectContext.get())
                    .map(ConnectContext::getExecTimeout)
                    .orElse(Config.insert_load_default_timeout_second)
```